draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -177,57 +177,127 @@ $scene-illustration-size-mobile: 200px;
 .inline,
 .stacked {
   flex-direction: row;
+  align-items: unset;
   gap: $spacing-md;
+  min-width: unset;
+  flex-wrap: wrap;
 
-  @include ca-media-mobile {
-    flex-direction: column;
+  @include ca-media-tablet-and-under {
+    flex-direction: unset;
   }
 
-  .illustrationWrapper {
-    padding: 0;
+  &.stackButton {
```

Comment:
Is the extra nesting necessary? CSS modules will actually ensure your classes differ between scss files (eg. `.stacked` here is not compiled to be the same as `.stacked` elsewhere). If you need the new class, it does not necessarily need to live within a parent.

Also, at some point, if there's a significant level of variation (especially since this file seems very long), we might be better making them separate components to keep them cleaner (I don't have full context though, so I could be wrong).

Reply:
Ideally, we create the new Tile component however, given the time urgency to deliver the project we've collectively decided to leverage off the Guidance Block until the Tile component is created acknowledging the tech debt. At that point we'll switch out. 

Since the variations rely on many existing functionality in the Guidance block, it didn't make sense to create entirely new separate components for this. Hence, there's overrides and nested CSS to override the existing styles.

Reply:
You should still be able to create the variations without nesting usually :)
Sure if there has been discussion around the approach and it's urgent then it's understandable. Since I wasn't part of the discussion earlier, was the conclusion that this component will be deprecated and split into different components based on usage? Or perhaps just refactored since it looks somewhat old (class based as opposed to a function component).

Reply:
Pushed up a change reducing a bunch of classnames and simplify to only use the `centerContent` for adjusting content when the card is due to show the centered content.